source/Core/Inc/Settings.h

```diff
@@ -41,6 +41,8 @@ typedef struct {
   uint8_t descriptionScrollSpeed : 1; // Description scroll speed
   uint8_t lockingMode : 2;            // Store the locking mode
   uint8_t KeepAwakePulse;             // Keep Awake pulse power in 0.1 watts (10 = 1Watt)
+  uint8_t KeepAwakePulseWait;         // Time between Keep Awake pulses in 2500 ticks
```

Comment:
Like above, can we swap tick units to ms or seconds as appropriate?

Reply:
Agreed.